lib/ansible/modules/packaging/os/pulp_repo.py

```diff
@@ -84,6 +109,15 @@
     description:
       - Repo plugin type to use (i.e. C(rpm), C(docker)).
     default: rpm
+  repoview:
+    description:
+      - Whether to generate repoview files for a published repository. Setting
+        this to "yes" automatically activates `generate_sqlite`.
+    required: false
+    type: bool
+    choices: ["yes", "no"]
```

Comment:
I don't think that choices are appropriate on a boolean option. I don't see them used on other modules on boolean types. I believe the documentation generator will handle this automatically. I think that the defaults of no should also be quoted (at least by precedent here: https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/files/file.py#L63).